MMM-TeslaFi.js

```diff
@@ -254,6 +254,36 @@ Module.register("MMM-TeslaFi", {
             `;
           }
           break;
+
+        case "state":
+          table += `
+					<tr>
+						<td class="icon"><span class="zmdi zmdi-hc-fw ${
+              t.carState === "Sentry"
```

Comment:
I'm not sure I understand the logic here, could you explain? From my understanding, this will show a 'sentry' indication if sentry mode is enabled, but otherwise it'll always show a 'P' symbol for parked, even if the vehicle is driving? 

If that's the case, then shouldn't we add an icon for other vehicle states as well,?

Reply:
Agreed, will try to find an appropriate icon and add it for driving